nntrainer/utils/util_func.h

```diff
@@ -23,6 +23,10 @@
 #ifndef __UTIL_FUNC_H__
 #define __UTIL_FUNC_H__
 
+#ifdef _WIN32
+#define MAX_PATH_LENGTH 1024
```

Comment:
I wonder where it is used?

Reply:
It is used in "_fullpath" function at "util_func.cpp"(line 202). Would it be better to move this definition to a cpp file?

Reply:
> It is used in "_fullpath" function at "util_func.cpp"(line 202). Would it be better to move this definition to a cpp file?

I think it is. Thanks.